File: Stick/Assets/Scripts/CameraControl.cs
using UnityEngine;
using System.Collections;

public class CameraControl : MonoBehaviour {//Controlling camera when you alive
	public PlayerController player;
	public bool isFollowing;

	public float xOffset;
	public float yOffset;
	void Awake(){
		DontDestroyOnLoad (this.gameObject);
	}
	// Use this for initialization
	void Start () {
		player = FindObjectOfType<PlayerController> (); //Finding player

		isFollowing = true;
	}
	
	// Update is called once per frame
	void Update () {
		if (isFollowing) {
			transform.position = new Vector3(player.transform.position.x + xOffset, player.transform.position.y + yOffset, player.transform.position.z - 2f);
		}
	}
}

File: 1stRPG/Assets/Script/Camera Control.js
#pragma strict
var Target: GameObject;
var speed: float;
var TargetPos: Vector3;
function Start () {

}

function Update () {
	TargetPos = new Vector3(Target.transform.position.x, Target.transform.position.y, -10);
	transform.position = Vector3.Lerp (transform.position, TargetPos, speed *  Time.deltaTime);
}